bundles/framework/admin-announcements/view/AnnouncementsHelper.js

```diff
@@ -0,0 +1,87 @@
+export const announcementsHelper = () => {
+    const getAdminAnnouncements = (handler) => {
+        if (typeof handler !== 'function') {
+            return;
+        }
+
+        jQuery.ajax({
+            type: 'GET',
+            dataType: 'json',
+            url: Oskari.urls.getRoute('GetAdminAnnouncements'),
```

Comment:
Consider just having a parameter if you want to get all the announcements for admin vs just the current ones for regular users instead of having separate action routes.